docs/docs/topics/schema.mdx

```diff
@@ -14,7 +15,61 @@ Out of the box, Infrahub doesn't have a schema for most things and it's up to us
 
 Unlike traditional databases that can only have one schema at a time, in Infrahub it is possible to have a different schema per branch. This is possible because the schema itself is stored in the database like any other object.
 
-There are several ways to [load a new schema](/guides/import-schema).
+## Schema file
+
+The recommended way to manage and load a schema is to create a schema file in YAML format. With a schema file it's possible to:
+
+- Define new nodes
+- Extend nodes, by adding attributes or relationships to the existing nodes
+
+At a high level, the format of the schema file looks like the following:
+
+```yaml
+---
+version: '1.0'
+nodes:
+    - <new nodes are defined here>
+extensions:
+  nodes:
+    - <node extensions are defined here>
+```
+
+<details>
+  <summary>Example of schema file that is defining new nodes and adding a relationship to an existing one</summary>
+  <CodeBlock language="yaml">{import_infrastructureExtensionRackYaml}</CodeBlock>
+</details>
+
+:::note
+
+To help with the development process of a schema definition file, you can leverage [schema validation](/reference/schema-validation) within your editor.
+
+:::
+
+## Load a schema file
```

Comment:
Maybe here we could guide people in the choice, when to use git integration vs infrahubctl, pros and cons.

For instance I added that paragraph in the schema library:
```
There are several ways to [load a schema in Infrahub](https://docs.infrahub.app/guides/import-schema):

Quick View: To take a quick look at a schema, you can use Infrahub CTL. Follow [this guide](https://docs.infrahub.app/infrahubctl) to install Infrahub CTL.
Controlled Integration: For a more organized and unified approach to loading schemas, you can connect a Git repository. Follow [this guide](https://docs.infrahub.app/guides/repository) to connect a Git repository.
```